crates/iroha_multisig_data_model/src/lib.rs

```diff
@@ -22,10 +37,10 @@ pub struct MultisigAccountArgs {
     /// </div>
     // FIXME #5022 prevent multisig monopoly
     // FIXME #5022 stop accepting user input: otherwise, after #4426 pre-registration account will be hijacked as a multisig account
-    pub account: PublicKey,
-    /// List of accounts and their relative weights of responsibility for the multisig
+    pub account: AccountId,
+    /// List of signatories and their relative weights of responsibility for the multisig account
     pub signatories: BTreeMap<AccountId, Weight>,
-    /// Threshold of total weight at which the multisig is considered authenticated
+    /// Threshold of total weight at which the multisig account is considered authenticated
     pub quorum: u16,
     /// Multisig transaction time-to-live in milliseconds based on block timestamps. Defaults to [`DEFAULT_MULTISIG_TTL_MS`]
     pub transaction_ttl_ms: u64,
```

Comment:
```suggestion
    pub transaction_ttl_ms: NonZeroU64,
```

Reply:
0 is not semantically well defined. Does it mean we should reject `MultisigRegister` immediately? if so, then you can keep 0

Reply:
Ok, nonzero would save users from finding any particular meaning in 0. The same for quorum.

Regarding weight, I'd allow 0 as they might want to include observers who don't join governance